onebusaway-gtfs-transformer/src/main/java/org/onebusaway/gtfs_transformer/impl/VerifyFutureRouteService.java

```diff
@@ -90,15 +93,21 @@ public void run(TransformContext context, GtfsMutableRelationalDao dao) {
 
         _log.info("Active routes {}: {}, {}: {}, {}: {}",
                 tomorrow, tripsTomorrow, nextDay, tripsNextDay, dayAfterNext, tripsDayAfterNext);
+        es.publishMetric(AwsContextService.getNamespace(), "RoutesContainingTripsTomorrow", "feed", feed, tripsTomorrow[0]);
+        es.publishMetric(AwsContextService.getNamespace(), "RoutesMissingTripsFromAtisButInRefTomorrow", "feed", feed, tripsTomorrow[1]);
```

Comment:
if you are using 0 and 1 to indicate today/tomorrow, its customary to replace 0 with a constant TODAY.  Otherwise known as MAGIC NUMBERS are BAD.